axelor-sale/src/main/resources/domains/SaleConfig.xml

```diff
@@ -32,6 +32,10 @@
     <decimal name="defaultHoldBackPercentage" title="Default holdback percentage"/>
     <many-to-one name="saleOrderPrintTemplate" title="Sale order template"
       ref="com.axelor.apps.base.db.PrintingTemplate"/>
+    <integer name="orderCreationModeSelect"
```

Comment:
please add "cart" in the name, so maybe `cartOrderCreationModelSelect`, also do it for the name of the selection.